File: 0/Libraries/Test/Intense.TestApp/ViewModels/DisplayableTheme.cs
using Intense.Presentation;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Windows.UI.Xaml;

namespace Intense.TestApp.ViewModels
{
    /// <summary>
    /// Represents a display theme.
    /// </summary>
    public class DisplayableTheme
        : Displayable
    {
        /// <summary>
        /// Initializes a new instance of the <see cref="DisplayableTheme"/>.
        /// </summary>
        /// <param name="displayName"></param>
        /// <param name="theme"></param>
        public DisplayableTheme(string displayName, ApplicationTheme theme)
        {
            this.DisplayName = displayName;
            this.Theme = theme;
        }

        /// <summary>
        /// Gets the them.
        /// </summary>
        public ApplicationTheme Theme { get; }
    }
}

File: 0/Templates/VisualBasic/ProjectTemplates/SplitViewAppVBTemplate/Presentation/DisplayableTheme.vb
Imports Intense.Presentation

Namespace Presentation
    Public Class DisplayableTheme
        Inherits Displayable

        Private _theme As ApplicationTheme

        Public Sub New(displayName As String, theme As ApplicationTheme)
            Me.DisplayName = displayName
            _theme = theme
        End Sub

        Public ReadOnly Property Theme As ApplicationTheme
            Get
                Return _theme
            End Get
        End Property
    End Class
End Namespace
